zavod/zavod/validators/assertions.py

```diff
@@ -0,0 +1,74 @@
+from typing import Dict, Any, Optional, cast
+from zavod.context import Context
+
+from zavod.entity import Entity
+from zavod.meta.assertion import Assertion, Comparison, Metric
+from zavod.exporters.statistics import Statistics
```

Comment:
this feels a tad icky, but rather than move the Statistics object somewhere more specific, I think once we want more fancy assertions, we'll build a dedicated aggregator and leave the exporter alone.

Reply:
no I like this a lot. Let's split it up early. The stats is part of our exposed interface so separating them makes a ton of sense to get ease of change for the validator.

Reply:
Do you mean I should implement statistics for the validator now, or use it as implemented here until we want a more powerful validator?